packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -337,6 +365,19 @@ module.exports = function (log, config, oauthdb) {
     return constructLocalDateString(timeZone, translator.language, date);
   };
 
+  Mailer.prototype._getLocalizedCurrencyString = function (
+    amountInCents,
+    currency,
+    acceptLanguage
+  ) {
+    const translator = this.translator(acceptLanguage);
```

Comment:
Please forgive my lack of context around this code, but I just want to check: you noted below that acceptLanguage is not just a single locale, so does this always return a single locale (or null) for Intl.NumberFormat to receive?

Reply:
Yeah, this is the bit that negotiates a single locale from the acceptLanguages string. It's basically the reason why we have these wrapper functions in the module, which I discovered during development :)